test/notifications/notification_server_test.exs

```diff
@@ -300,16 +302,18 @@ defmodule Notifications.NotificationServerTest do
 
     test "broadcasts, saves to the DB, and logs new notifications for waivers with recognized reason for vehicles on selected routes",
          %{user: user} do
+      vehicle = @vehicle
```

Comment:
question (non-blocking): Why is this assignment needed?

Reply:
From how I understand it, any invocation of a module attribute is expanded inline at each point. So to [avoid expanding this somewhat large map all over the place](https://github.com/mbta/skate/pull/2102/files/27e69efac8647e966a57d8cb02f02a874fb81fe6#diff-5ca46ffdee1a00ab30040cca268ee561ffc4aa61a4533dcf066372f54842b50cR315-R316) I've bound it to a local variable for field access. I also did this in hopes that we could convert this to use factories in the future.

```suggestion
      vehicle = build(:vehicle)
```

Reply:
https://hexdocs.pm/elixir/Kernel.html#@/1-attention-multiple-references-of-the-same-attribute